extensions/extension.go

```diff
@@ -0,0 +1,34 @@
+package extensions
+
+import (
+	"math"
+
+	"grol.io/grol/object"
+)
+
+func init() {
```

Comment:
I would remove the init that relies on magical code/hack

See
https://github.com/grol-io/grol/pull/96#discussion_r1702804068